app/components/game-card.js

```diff
@@ -30,6 +36,30 @@ export default Ember.Component.extend({
   /** @property {Boolean} only interact with the cards if you're player one */
   readOnly: false,
 
+  /** Attach document key handlers for magnifying purposes. */
+  attachKeyHandlers: Ember.on('init', function () {
```

Comment:
sorry didn't mean to say whether or not you clicked down on the game-card but if you keyed down or keyed up on the game-card